spec/index.html

```diff
@@ -1516,58 +1516,74 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
+              there is an entry <var>e<sub>b</sub></var> in |b|
+              such that the <a data-cite="INFRA#map-key">key</a> in <var>e<sub>a</sub></var>
+              equals the <a data-cite="INFRA#map-key">key</a> in <var>e<sub>b</sub></var>,
+              and the <a data-cite="INFRA#map-value">value</a> in <var>e<sub>a</sub></var>
+              equals the <a data-cite="INFRA#map-value">value</a> in <var>e<sub>b</sub></var>.</li>
+          </ul>
+      </dd> 
+
+      <dt id="JSON-mapping">The <a>lexical-to-value mapping</a></dt>
+      <dd>maps every element of the lexical space to the result of parsing it into a
+        <a>string</a>,
+        number (<a data-cite="XMLSCHEMA11-2#double"><strong>xsd:double</strong></a>),
+        <a data-cite="INFRA#ordered-map">map</a>,
+        <a data-cite="INFRA#list">list</a>, or
+        literal value (<a data-cite="INFRA#boolean">`true`, `false`</a>, and <a data-cite="INFRA#nulls">null</a>).
+
         <ul>
-          <li>They MUST NOT contain any unnecessary whitespace.</li>
-          <li>Keys in objects MUST be ordered lexicographically.</li>
-          <li>Native Numeric values MUST be serialized according to
-            <a data-cite="ECMASCRIPT#sec-tostring-applied-to-the-number-type">Section&nbsp;7.1.12.1</a>
-            of [[ECMASCRIPT]].</li>
-          <li>Strings MUST be serialized with Unicode code points from <code class="codepoint">U+0000</code>
-            through <code class="codepoint">U+001F</code> using lower case hexadecimal Unicode notation
-            (`\uhhhh`) except for the set of predefined JSON control characters —
-            `U+0008`, `U+0009`, `U+000A`, `U+000C`, and `U+000D` — which SHOULD be
-            serialized as `\b`, `\t`, `\n`, `\f`, and `\r`, respectively. All
-            other Unicode characters SHOULD be serialized "as is", except
-            `U+005C` (`\`) and `U+0022` (`"`), which SHOULD be serialized as
-            `\\` and `\"`, respectively.</li>
+          <li>A <a href="RFC8259#section-4">JSON Object</a> is mapped to a <a data-cite="INFRA#ordered-map">map</a>
+            by transforming each object member into a <a data-cite="INFRA#map-entry">map entry</a>
+            with the <a data-cite="INFRA#map-key">key</a> taken from the member name and
+            <a data-cite="INFRA#map-value">value</a> taken by performing this mapping
+            to the member value.
+            <a data-cite="INFRA#map-entry">Map entries</a> are treated as being unordered.
+            </li>
+          <li>A <a href="RFC8259#section-5">JSON Array</a> is mapped to a <a data-cite="INFRA#list">list</a>
+            by performing this mapping on each array value.</li>
+          <li>A <a href="RFC8259#section-6">JSON Number</a> is mapped to
+            an <a data-cite="XMLSCHEMA11-2#double"><strong>xsd:double</strong></a>
+            using a method consistent with <a data-cite="XMLSCHEMA11-2#f-doubleLexmap">doubleLexicalMap</a>.</li>
```

Comment:
This note addresses @pfps's concerns about number values potentially being outside of the lexical space for JSON numbers.

```suggestion
            using a method consistent with <a data-cite="XMLSCHEMA11-2#f-doubleLexmap">doubleLexicalMap</a>.
            <div class="note">Some numbers cannot be represented
              as finite <a data-cite="XMLSCHEMA11-2#double"><strong>xsd:double</strong></a> values
              and may map to `+INF` or `-INF`.</div>
          </li>
```